src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -44,51 +44,49 @@
         </a>.
 
       </remarks>
-    </member>
-    <example name="TensorflowTransform">
       <example>
         <code language="csharp">
           pipeline.Add(new TextLoader(dataFile).CreateFrom&lt;MNISTData&gt;(useHeader: false));
           pipeline.Add(new ColumnCopier(("NumericImageVec", "Input");
           pipeline.Add(new TensorFlowScorer()
           {
-          ModelFile = model_location;
-          InputColumns = new []{ &quot;Input&quot; };
-          OutputColumns = &quot;Output&quot;
+            ModelFile = model_location;
+            InputColumns = new []{ "Input" };
+            OutputColumns = "Output"
           }
         </code>
       </example>
       <example>
         <code language="csharp">
           var pipeline = new LearningPipeline(seed: 1);
           pipeline.Add(new TextLoader(dataFile).CreateFrom&lt;CifarData&gt;(useHeader: false));
-          pipeline.Add(new ImageLoader((&quot;ImagePath&quot;, &quot;ImageReal&quot;))
+          pipeline.Add(new ImageLoader(("ImagePath", "ImageReal"))
           {
-          ImageFolder = imageFolder
+            ImageFolder = imageFolder
           });
 
-          pipeline.Add(new ImageResizer((&quot;ImageReal&quot;, &quot;ImageCropped&quot;))
+          pipeline.Add(new ImageResizer(("ImageReal", "ImageCropped"))
```

Comment:
>" [](start = 54, length = 1)

Turns out that " in the <code> sections are ok, but not the <, > characters. Converted them to make the example more readable.